src/data/items.js

```diff
@@ -24,12 +24,14 @@ const {
 } = cropType
 const { SEED, GROWING } = cropLifeStage
 const {
+  CHOCOLATE,
   COW_FEED,
   CROP,
   FERTILIZER,
   HUGGING_MACHINE,
   MILK,
   SCARECROW,
+  SOY_MILK,
```

Comment:
Based on my comments about these being defined, I think these changes can be reverted.